determined-ai/determined
# feat: parallel coordinates chart on trial hyperparameters tab [WEB-1196]

## Description
Task
Update the viz on the single trial “Hyperparameters” tab for single trials that are part of a multi-trial search from current vis to pre-filtered parallel coordinates chart Show PC chart.

Result
![Screenshot 2023-05-19 at 15-24-24 Trial 5820 - Determined](https://github.com/determined-ai/determined/assets/15078396/bfa6c6b7-348c-4600-8b79-dca2a4be3911)






## Test Plan

- Navigate to a Trial Details page inside a multi-trial experiment
- Go to Hyperparameters tab
- It should look like the screenshot above, with the current trial line being active and all others faded but visible
- Chart should **not** be interactable (ie reordering dimensions and creating filters), but all selects in chart header should work as normal
- Below chart there should be a table showing exact values of current trial hyperparameters (the list of hps might differ from the chart because constant hyperparameters are not shown in parallel coordinates)





## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
